Assets/SC_bALL.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_bALL : MonoBehaviour
+{
+    private Transform _transform;
+    public bool thisvarshowsifanobjectisgrounded; //check if an object is afloat or not
+    void Start()
+    {
+        _transform = GetComponent<Transform>();
```

Comment:
There is no need to initialize the Transform using the Start() method and the GetComponent() method. Instead, it will be possible to get a Transform object anywhere in the class using gameObject.transform